### ISSUE TITLE
Reproducible exports

### ISSUE THREAD

**Issue by SergioCasca** [Microsoft/VoTT#348]
An option to set a seed on the extract tags menu. For example, applying consecutive extractions for CNTK on the same image set with the same tags returns distinct positive and testImages sets.

**Comment by Legor** [comment]
I dont think this is easily done with the current implementation since the algorithm doesn't use a seed. It's not like there is PSNG used to create the test/training indices or i dont see it. If anyone is interested, the relevant code is inside here:
https://github.com/Microsoft/VoTT/blob/ec6057c4c95780f7547d5c55245c6f48b396e29c/src/lib/detection_algorithms/detectionUtils.js#L1-L15

Probably what you want to do is to update the tags of a previously generated training/test split? So maybe its a better solution to implement this option ("Update with new tags") or sth. like that.
